code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from "react-router-dom";
+
+import "components.css/movieDetails.css"
+
+//fetch according to movie id 
+//this is the page that shows one movie with its details
+
+export const MovieDetails = () => {
+  const {id} = useParams();
+
+  const [movieDetails, setMovieDetails] = useState({});
+  
+  const getMovieDetails = (movieId) => {
+    const MOVIE_DETAILS_URL = `https://api.themoviedb.org/3/movie/${movieId}?api_key=636ed5609cbdb1c9a92992698d99fea5`;
+    fetch(MOVIE_DETAILS_URL)
+      .then(response => response.json())
+      .then(json => {
+        setMovieDetails(json)
+      })
+  }
+
+  useEffect (()=> {
+    getMovieDetails(id)
+  },[id]);  
+  console.log(movieDetails);
+
+  return(
+    <article className="movie-details"> 
```

Comment:
Here you could add the return button as shown in the example by Technigo. I did like this (but I*m sure there are other ways as well 😀)

`<Link className="homepage-link" to="/">
        <img className="back-button" src="/back-button.svg" alt="Back Button" />
        <p>
          Movies
        </p>
      </Link>`